manifests/service.pp

```diff
@@ -15,4 +15,11 @@
     }
   }
 
+  $use_servers = join($servers, " ")
+  if $autosync {
+    exec { "/usr/sbin/ntpdate -u -s ${use_servers}":
+     refreshonly => true
```

Comment:
This really is only important when we are starting the NTP service and there is a massive difference in time, NTP will complain.  If the NTP service is running there will be no time creep.

So this is only triggered in service start.